File: QE-SELINIUM JAVA/demo/src/main/java/demos/Activity2.java
package demos;
import org.openqa.selenium.By;
import org.openqa.selenium.WebElement;
import org.openqa.selenium.firefox.FirefoxDriver;

public class Activity2 {

    public static void main(String[] args) {

        // Launch browser
        FirefoxDriver driver =new FirefoxDriver();
        // Open URL
        driver.get("https://training-support.net/webelements/login-form/");

        // Print page title
        System.out.println("Page Title: " + driver.getTitle());

        // Find username field and enter value
        WebElement username = driver.findElement(By.id("username"));
        username.sendKeys("admin");

        // Find password field and enter value
        WebElement password = driver.findElement(By.id("password"));
        password.sendKeys("password");

        // Find login button and click
        WebElement loginButton = driver.findElement(By.xpath("//button[text()='Submit']"));
        loginButton.click();

        // Close browser
        driver.quit();
    }
}

File: QE-SELENIUM-python/Activity2.py
from selenium import webdriver
from selenium.webdriver.common.by import By

with webdriver.Firefox() as driver:

    # Launch browser
    driver=webdriver.Firefox()

    # Open URL
    driver.get("https://training-support.net/webelements/login-form/")

    # Print page title
    print("Page Title:", driver.title)

    # Enter username
    username = driver.find_element(By.ID, "username")
    username.send_keys("admin")

    # Enter password
    password = driver.find_element(By.ID, "password")
    password.send_keys("password")

    # Click login button
    login_button = driver.find_element(By.XPATH, "//button[text()='Submit']").click()

    # Close browser
    driver.quit()

    #xpath //tagname[@attribute='value']